assets/js/commento.js

```diff
@@ -204,7 +207,7 @@
 
     var _showdownUrl = "/assets/showdown.min.js";
```

Comment:
We should split the front-end from the back-end, at least at a tree level. Spectre and showdown are both vendored files because they are external dependencies. In the future we may build those libraries ourselves. We should have a `public` folder like gogs.